### PR TITLE
Update `imp_helm_legend.esp` KAP WACCF patches

### PR DESCRIPTION
**Pull request by loicreynier**
[kryptopyr's Patch Hub](https://www.nexusmods.com/skyrimspecialedition/mods/19518) now includes new WACCF/AMB patches with the Beard Mask Fix (BMF).

### CODE REVIEW

**Comment by loicreynier** [comment]
I'll try to use a regex to shorten the condition.

**Comment by loicreynier** [comment]
This regex should match all the relevant patches from KAP:
- `imp_helm_legend_WACCF_Patch.esp`
- `imp_helm_legend_WACCF_AMB_Patch.esp`
- `imp_helm_legend_BMF_WACCF_Patch.esp`
- `imp_helm_legend_BMF_WACCF_AMB_Patch.esp`

**Review by MacSplody — APPROVED** [review]

File: masterlist.yaml [hunk 1]
```diff
@@ -18363,7 +18363,7 @@ plugins:
     after: [ 'Circlets and Masks with all Robes and Hoods.esp' ]
     msg:
       - <<: *patch3rdParty_KPH_WACCF
-        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_WACCF_Patch.esp") and not active("imp_helm_legend_WACCF_AMB_Patch.esp")'
+        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_(BMF_)?WACCF_(AMB_)?_Patch\.esp")'
```
Comment (MacSplody) [inline masterlist.yaml:18366]: Sorry missed that extra `_` in my review.
```suggestion
        condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and not active("imp_helm_legend_(BMF_)?WACCF_(AMB_)?Patch\.esp")'
```

**Review by MacSplody — CHANGES_REQUESTED** [review]
Sorry missed that extra `_` in my review.

**Review by MacSplody — APPROVED** [review]

**Comment by loicreynier** [comment]
Whoops, thanks good catch.

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**